src/evidently/ui/api/projects.py

```diff
@@ -207,7 +214,7 @@ def get_snapshot_data(
     project_manager: Annotated[ProjectManager, Dependency(skip_validation=True)],
     log_event: Callable,
     user_id: UserID,
-) -> Response:
+) -> str:
```

Comment:
We lose media-type in this approach, maybe need to look at `-> Response[str]` so we can customize response as was before.